### DISCUSSION TITLE
Can we delete a repository in deleted repositories?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51090] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to delete any repos in [deleted repositories](https://github.com/settings/deleted_repositories) ?

**Comment by Fo00oX** [comment]
Seems like you cant do really much

but found this:
As mentioned in [About repository restoration](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#about-repository-restoration)

> A deleted repository can be restored within 90 days, unless the repository was part of a fork network that is not currently empty.

So that list is likely to be cleared out after 90 days for a given repository.
If that is too long, you would need to contact [GitHub support](https://support.github.com/request?tags=docs-generic).
